cypress/integration/regions_test/regions_test.js

```diff
@@ -0,0 +1,177 @@
+describe("Regions Test", function(){
+    beforeEach(function(){
+        cy.visit("/biotestmine/regions");
+    });
+
+    it("can search a gene region given by example", function(){
+        cy.get(".input-section").within(() => {
+            cy.contains("Show Example").click();
+            cy.get("textarea").should("include.text","MAL");
+            cy.get("button").filter(':contains("Search")').click();
+        })
+        cy.get(".results").within(() => {
+            cy.get(".single-feature").its("length").should("be.gt",0);
+            cy.contains("exon.46111").click();
+        })
+        cy.url().should("include","/report");
+    });
+
+    it("can type in chromosome coordinates and select features", function(){
+
+        cy.get(".input-section").within(() => {
+            cy.get("textarea").type("MAL1:0..100000",{delay:100});
+            cy.get('button[class="btn dropdown-toggle"]').click();
+            cy.contains("P. falciparum 3D7").click();
+            cy.contains("Features to include").children().click();
+            cy.contains("Gene").children().click();
+            cy.get("button").filter(':contains("Search")').click();
+        })
+        cy.get(".results").within(() => {
+            cy.get(".single-feature").its("length").should("be.gt",0);
+            cy.contains("Next").click();
+            cy.get(".single-feature").its("length").should("be.gt",0);
+            cy.get(".single-feature").first().click();
+        })
+        cy.url().should("include","/report");
+    });
+
+    it("can search regions with interbase coordinates", function(){
+        cy.get(".input-section").within(() => {
+            cy.get("textarea").clear().type("MAL1:0..30000",{delay:100});
+            cy.get(".radio-group").within(() => {
+                cy.contains("interbase").within(() => {
+                    cy.get(".check").should("not.be.visible");
+                    cy.get(".circle").should("be.visible").click();
+                    cy.wait(500);
+                    cy.get(".check").should("be.visible");
+                })
+            })
+            cy.get("button").filter(':contains("Search")').click();
+        })
+        cy.get(".results").within(() => {
+            cy.get("#region-result-0 > span").should("include.text","MAL1 1..30000");
+        })
+    })
+
+    it("can select to perform strand-specific region search", function(){
```

Comment:
Can you extend this test to also search the reverse strand, meaning
```
MAL1:0..30000
MAL1:30000..0
```
and checking that the second result includes arrow-left.